internal/pkg/aws/cloudwatch/cloudwatch.go

```diff
@@ -125,6 +133,58 @@ func (cw *CloudWatch) AlarmStatuses(opts ...DescribeAlarmOpts) ([]AlarmStatus, e
 	return alarmStatuses, nil
 }
 
+// AlarmDescriptions returns the config of alarms filtered by name.
+func (cw *CloudWatch) AlarmDescriptions(alarmNames []string) ([]*AlarmDescription, error) {
+	var alarmDescriptions []*AlarmDescription
+	in := &cloudwatch.DescribeAlarmsInput{
+		AlarmNames: aws.StringSlice(alarmNames),
+	}
+	for {
+		alarmResp, err := cw.client.DescribeAlarms(in)
```

Comment:
Should this be outside of for loop? Or am I misunderstanding something here? Are we describing all the alarms in every loop?

Reply:
Good question! It's because of the `NextToken`-- if there are multiple pages of output, then we have to paginate through it.